docs/rules/regex-shorthand.md

```diff
@@ -27,3 +25,12 @@ const regex = /\W/;
 const regex = /\W/i;
 const regex = /\d\.\w\-\D/i;
 ```
+
+## Options
+
+### disableCharacterSorting
+
+Type: `boolean`\
+Default: `false`
+
+Disables optimizations that affects the sorting of characters.
```

Comment:
The description here needs to be clearer about what exactly will be sorted. "characters" is too ambiguous.

Reply:
@sindresorhus There are two options as I see it:

1. Get into the specifics of regexp-tree and be up front about the disabling of `charClassClassrangesMerge`, which the sorting is a needed side-effect of. This is probably too implementation specific and doesn't convey the purpose clearly.
2. Clearly convey the intended effect by naming it `sortCharacters`/`sortCharacterClasses` (which the description of maybe could be spelled out in a somewhat more specific manner, but I have a hard time pinpointing that, maybe due to me not being the best at Regexp terminology).

I lean towards the second and I get the impression that you do as well?

Reply:
Sorry, missed this. Yes, `2`. Could you also add an example? Not everyone knows what character classes are.